[2023-09-02 19:19:32,105] INFO [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.1.6:5000 - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:19:32,107] INFO [33mPress CTRL+C to quit[0m - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:19:41,202] INFO 127.0.0.1 - - [02/Sep/2023 19:19:41] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:19:41,382] INFO 127.0.0.1 - - [02/Sep/2023 19:19:41] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:19:41,391] INFO 127.0.0.1 - - [02/Sep/2023 19:19:41] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:19:41,392] INFO 127.0.0.1 - - [02/Sep/2023 19:19:41] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:19:41,639] INFO 127.0.0.1 - - [02/Sep/2023 19:19:41] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:19:41,652] INFO 127.0.0.1 - - [02/Sep/2023 19:19:41] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:19:51,946] ERROR Exception on /player/2 [GET] - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\flask\app.py L1414
Traceback (most recent call last):
  File "C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\flask_openapi3\scaffold.py", line 127, in view_func
    response = func(**func_kwargs)
               ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\app.py", line 36, in get_player_by_id
    return service.find_by_id(path.id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\service\player_service.py", line 12, in find_by_id
    player = session.query(Player).filter(player.id == id).first()
                                          ^^^^^^
UnboundLocalError: cannot access local variable 'player' where it is not associated with a value
[2023-09-02 19:19:51,956] INFO 127.0.0.1 - - [02/Sep/2023 19:19:51] "[35m[1mGET /player/2 HTTP/1.1[0m" 500 - - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:20:29,544] INFO [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.1.6:5000 - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:20:29,545] INFO [33mPress CTRL+C to quit[0m - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:20:31,447] INFO Player encontrado: {player} - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\service\player_service.py L19  
[2023-09-02 19:20:31,452] INFO 127.0.0.1 - - [02/Sep/2023 19:20:31] "GET /player/2 HTTP/1.1" 200 - - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:50:30,111] INFO [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.1.6:5000 - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:50:30,112] INFO [33mPress CTRL+C to quit[0m - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:50:32,647] INFO 127.0.0.1 - - [02/Sep/2023 19:50:32] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:50:32,851] INFO 127.0.0.1 - - [02/Sep/2023 19:50:32] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:50:32,859] INFO 127.0.0.1 - - [02/Sep/2023 19:50:32] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:50:32,861] INFO 127.0.0.1 - - [02/Sep/2023 19:50:32] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:50:33,141] INFO 127.0.0.1 - - [02/Sep/2023 19:50:33] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
[2023-09-02 19:50:33,144] INFO 127.0.0.1 - - [02/Sep/2023 19:50:33] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\thymont\OneDrive\Documentos\Engenharia de software\mvp-3-card-game-backend\app\venv\Lib\site-packages\werkzeug\_internal.py L187 
